acbuild/end.go

```diff
@@ -44,16 +46,13 @@ func runEnd(cmd *cobra.Command, args []string) (exit int) {
 		cmd.Usage()
 		return 1
 	}
-	if len(args) > 1 {
-		stderr("end: incorrect number of arguments")
-		return 1
-	}
 
 	if debug {
 		stderr("Ending build. Writing completed ACI to %s", args[0])
 	}
 
-	err := lib.End(tmpacipath(), args[0], path.Join(contextpath, workprefix), overwrite)
+	err := lib.End(tmpacipath(), args[0], path.Join(contextpath, workprefix),
+		overwrite, sign, args[1:])
```

Comment:
Ehhh, I am wary of this approach because it precludes us from supplying other kinds of args. Can we have them as options passed to `--sign` rather than args[1:] ?